notebooks/7.Incorporating_Item_Metadata_Dataset_to_your_Predictor.ipynb

```diff
@@ -65,7 +69,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Although, we have set the region to us-west-2 below, you can choose any of the 6 regions that the service is available in."
+    "Configure the s3 bucket name and region name for this lesson.\n",
```

Comment:
nit : Capitalize S3. Sorry for bringing this up, so that it establishes one common pattern.

Reply:
Addressed by the latest commit.